data_science.md

```diff
@@ -4,6 +4,16 @@ This file contains resources to learn about Data Science and its sub fields.
 
 * Classical Machine Learning
 * Computer Vision
-* Deep Learning
+* [Deep Learning](#deep-learning)
 * Natural Language Processing 
 * Reinforcement Learning
+
+## Deep Learning
+
+### Neural Networks by 3Blue1Brown
+
+A playlist containing 4(four) videos about neural networks and their working. It gives a mathematical intution about neural networks and concepts like Gradient Desent and Backpropogation. 
+
+Website - https://www.youtube.com/playlist?list=PLZHQObOWTQDNU6R1_67000Dx_ZCJB-3pi
+
+![3blue1brown](https://user-images.githubusercontent.com/41947720/66421378-6161ce00-ea25-11e9-9128-10473cd5ccb4.png)
```

Comment:
Please link the image to the website.

Reply:
@aviaryan, thanks for the instruction, I have made the requested change.